app/lib/app/SecvisogramPage/View/FormEditor/schema.js

```diff
@@ -4521,6 +4524,7 @@ export default /** @type {const} */ ({
                             description: 'FullProductNameDescription',
                           },
                           addMenuItemsForChildObjects: true,
+                          uiType: 'WITH_GENERATED_PRODUCT_ID',
```

Comment:
Hmm - I think that one does not longer exist...

Reply:
Yes, I updated it.